chromium/browser_type.go

```diff
@@ -240,22 +239,26 @@ func (b *BrowserType) allocate(
 		return nil, err
 	}
 
-	path := opts.ExecutablePath
-	if path == "" {
-		path = b.ExecutablePath()
+	path, ok := ExecutablePath(opts.ExecutablePath, b.envLookupper, exec.LookPath)
```

Comment:
Should this also check whether the user-provided path exists? Currently, `NewLocalBrowserProcess` does that. But it might make more sense to encapsulate it in `ExecutablePath`—even before launching a process.

Reply:
I agree, i found it confusing that it didn't do that.

Reply:
If you make this change, then the `ErrChromeNotInstalled` could be returned from `ExecutablePath` instead of a `bool`.

Reply:
👍

Reply:
See 5499e10.